### ISSUE TITLE
Access to the EXTRA dataset audio

### ISSUE THREAD

**Issue by nystromer** [marl/medleydb#94]
I noticed that there is plenty more songs in the EXTRA dataset. I also noticed that there is a download script pointing to a private Google Drive. Is it possible to gain access to this extra data?

**Comment by carlthome** [comment]
I guess the tracks are copyrighted so it becomes non-trivial to share them, but easy access to EXTRA audio would be very useful in order to compare results to

>Gururani, Siddharth, Cameron Summers, and Alexander Lerch. "Instrument Activity Detection in Polyphonic Music using Deep Neural Networks." ISMIR. 2018.

which I believe uses the EXTRA tracks as the authors mention having 330 MTRs from MedleyDB but there's only 196 mentioned on https://medleydb.weebly.com/ (incl. V2).

**Comment by carlthome** [comment]
@SiddGururani, in the ISMIR 2018 paper for IAD, did you use V1, V2 and EXTRA tracks from MedleyDB? Is that why the paper mentions 330 tracks instead of just the 196 public ones? Is there a download link for the EXTRA tracks somewhere (couldn't find one on Zenodo)?

**Comment by rabitt** [comment]
Hi both - can you send me an email and we'll discuss there?

**Comment by SiddGururani** [comment]
@carlthome Yes, I did use all the tracks. I realized later that this is indeed tricky from a copyright standpoint.

**Comment by da-hon5** [comment]
@rabitt  hi, i'm also highly interested in the 'extra' and 'bach10' data. is it possible to give me access to these files? thanks a lot in advance :)

**Comment by chh-han** [comment]
can i get access to EXTRA datasets?
